homeassistant/components/telegram_bot/__init__.py

```diff
@@ -847,6 +872,34 @@ def send_location(self, latitude, longitude, target=None, **kwargs):
                 timeout=params[ATTR_TIMEOUT],
             )
 
+    def send_poll(
+        self,
+        question,
+        options,
+        is_anonymous,
+        allows_multiple_answers,
+        target=None,
+        **kwargs,
+    ):
```

Comment:
For a follow-up, it would be good to add type hints, based on the service schema.
eg. 
```python
    question: str,
    options: list[str],
    ...
    target: list[int] | None = None
```